x/rewards/abci.go

```diff
@@ -11,6 +11,18 @@ import (
 	"github.com/archway-network/archway/x/rewards/types"
 )
 
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
+
+	mintedTokens, found := k.GetInflationForRewards(ctx)
+	if found {
+		// Track inflation rewards
+		k.TrackInflationRewards(ctx, mintedTokens)
```

Comment:
if it's not found I'm not sure if skipping it is the correct way to do it.
Old params would be applied.